UPGRADE.md

```diff
@@ -17,6 +17,42 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 
 ## [From 7.0.0-alpha4 to Unreleased]
 
+### [shopsys/framework]
+#### PostgreSQL upgrade:
+We decided to move onto newer version of postgres to and you can too, even without losing your data.
+
+These steps are for migrating your data onto newer version of postgres and are inspired by [official postgres documentation](https://www.postgresql.org/docs/10/static/upgrading.html):
+
+If you are running your project natively then just follow these instructions, if you are using docker infrastructure you can follow steps written below.
+
+1.create a backup of your sql data by executing:
+
+`docker exec -it shopsys-framework-postgres pg_dumpall > your backupfile`
+
+1. copy new version of `docker-compose`:

```

Comment:
Format of list numbers is wrong - there are ones everywhere and it is not a list
https://github.com/shopsys/shopsys/blob/264fbc6b0415c783d03083bef93d98877cddd439/UPGRADE.md
please try to change it into a real list

Reply:
Okay, the problem was in indention of code blocks, after indenting them once everything works fine.